lib/parallel_tests/rspec/runner.rb

```diff
@@ -1,4 +1,5 @@
 require "parallel_tests/test/runner"
+require "rspec"
```

Comment:
prefer to not depend on rspec (to keep working with old/new versions of the gem) especially for trivial things like color-coding